src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
```

Comment:
What's the purpose of 8px here?

Reply:
I don't know what it's doing there at all. But it is definitely not needed there